packages/design-system-shared/CHANGELOG.md

```diff
@@ -7,12 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1]
+
+### Fixed
+
+- Bump @metamask/utils from 11.7.0 to 11.8.0 ([#827](https://github.com/MetaMask/metamask-design-system/pull/827))
+
```

Comment:
Updating MMDS shared with included PR to bump metamask utils. Other dependency updates not included/required for this package